### ISSUE TITLE
kubeadm v1.19 init phase addon: panic: runtime error: index out of range [0] with length 0

### ISSUE THREAD

**Issue by tstromberg** [kubernetes/minikube#9142]
Seen during cluster restarts with Kubernetes v1.19. It does eventually recover:

```
I0901 06:33:18.864862   99780 ssh_runner.go:148] Run: /bin/bash -c "sudo env PATH=/var/lib/minikube/binaries/v1.19.0:$PATH kubeadm init phase addon all --config /var/tmp/minikube/kubeadm.yaml"
I0901 06:33:19.072855   99780 retry.go:30] will retry after 110.466µs: /bin/bash -c "sudo env PATH=/var/lib/minikube/binaries/v1.19.0:$PATH kubeadm init phase addon all --config /var/tmp/minikube/kubeadm.yaml": Process exited with status 2
stdout:

stderr:
W0901 13:33:19.021097    4396 configset.go:348] WARNING: kubeadm cannot validate component configs for API groups [kubelet.config.k8s.io kubeproxy.config.k8s.io]
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns.isCoreDNSVersionSupported(0x1d5b6a0, 0xc000162b00, 0xc000716a00, 0xc000130f20, 0x153)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns/dns.go:413 +0x38f
k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns.createCoreDNSAddon(0xc0007e0000, 0x7fb, 0xb5f, 0xc00074a900, 0x309, 0x418, 0xc0004e6800, 0x20a, 0x3dc, 0x1d5b6a0, ...)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns/dns.go:271 +0x187
k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns.coreDNSAddon(0xc0004b6920, 0x1d5b6a0, 0xc000162b00, 0xc0006925fc, 0x0, 0x0)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns/dns.go:252 +0x505
k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns.EnsureDNSAddon(0xc0004b6920, 0x1d5b6a0, 0xc000162b00, 0x1d5b6a0, 0xc000162b00)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns/dns.go:109 +0x13e
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/init.runCoreDNSAddon(0x1a58420, 0xc000406420, 0xc000043540, 0xd)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/init/addons.go:92 +0x85
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).Run.func1(0xc00055f700, 0x0, 0x0)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow/runner.go:234 +0x1e8
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).visitAll(0xc0000b2a20, 0xc000539ce0, 0x0, 0x2)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow/runner.go:422 +0x6e
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).Run(0xc0000b2a20, 0xc00000da20, 0x0, 0x2, 0x726420, 0xc000539d48)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow/runner.go:207 +0x153
k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow.(*Runner).BindToCommand.func1.1(0xc0001bc000, 0xc00000da20, 0x0, 0x2, 0x0, 0x0)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow/runner.go:348 +0xda
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).execute(0xc0001bc000, 0xc00000da00, 0x2, 0x2, 0xc0001bc000, 0xc00000da00)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:842 +0x47c
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc000312000, 0xc000126000, 0x1cecec0, 0xc000126008)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:950 +0x375
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:887
k8s.io/kubernetes/cmd/kubeadm/app.Run(0x0, 0x18249a0)
	/workspace/anago-v1.19.0-rc.4.197+594f888e19d8da/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/app/kubeadm.go:50 +0x225
main.main()
	_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubeadm/kubeadm.go:25 +0x25
I0901 06:33:19.073112   99780 ssh_runner.go:148] Run: /bin/bash -c "sudo env PATH=/var/lib/minikube/binaries/v1.19.0:$PATH kubeadm init phase addon all --config /var/tmp/minikube/kubeadm.yaml"
```

**Comment by tstromberg** [comment]
This is being tracked upsttream at #2267

**Comment by prasadkatti** [comment]
@tstromberg - I think the link you added in the comment above is wrong

**Comment by sharifelgamal** [comment]
This is fixed in Kubernetes v1.19.2, which is part of minikube v1.13.1. Closing this.
